src/Microsoft.DocAsCode.Build.RestApi/RestApiHelper.cs

```diff
@@ -44,5 +45,15 @@ public static string FormatReferenceFullPath(string reference)
 
             throw new InvalidOperationException($"Full reference path \"{reference}\" must start with '/' or '#/'");
         }
+
+        public static string GetReferenceName(string formattedPath)
+        {
+            if (!formattedPath.StartsWith("#/") && !formattedPath.StartsWith("/"))
+            {
+                throw new ArgumentException($"Formatted reference path \"{formattedPath}\" must start with '/' or '#/'");
+            }
+
+            return formattedPath.Split('/').Last();
```

Comment:
before calling this method, formatted path has been decoded. But for better undestanding, I will merge this two methods into one.

---
In reply to: [69504557](https://github.com/dotnet/docfx/pull/464#discussion_r69504557) [](ancestors = 69504557)